lib/kube/proxy/forwarder.go

```diff
@@ -1502,6 +1502,9 @@ func (f *Forwarder) newClusterSessionLocal(ctx authContext) (*clusterSession, er
 	if !ok {
 		return nil, trace.NotFound("kubernetes cluster %q not found", ctx.kubeCluster)
 	}
+	if creds.tlsConfig == nil {
```

Comment:
At a glance this seems reasonable, but I'm more concerned about how we got a "bad" cred into the creds map in the first place. Maybe we should try and stop that from happening.